engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-publish.controller.js

```diff
@@ -30,17 +35,41 @@ angular.module('Bastion.content-views').controller('ContentViewPublishController
             $scope.working = false;
         }
 
+        $scope.calculateSolveDeps = function(contentViewSolveDep, solveDepSetting, skipSolveDep) {
```

Comment:
This isn't used in the view, but I attached it to the `$scope` so I could easily unit test a bunch of scenarios with it. It may not be best practice, but it gives me some good peace of mind knowing this logic is functioning as expected :peace_symbol: